randovania/data/json_data/prime3.txt

```diff
@@ -988,38 +1048,53 @@ Asset id: 10553270987934706862
   * Normal Door to Crash Site/Dock to G.F.S. Theseus (Door)
   > Dock to Falls of Fire
       Trivial
+  > Event - Satellite
+      Grapple Lasso
 
 > Dock to Crash Site (Tunnel); Heals? False
   * Morph Ball Door to Crash Site/Dock to G.F.S. Theseus (Tunnel)
   > Dock to Crash Site (Door)
       Trivial
```

Comment:
- Regular door node doesn't need to specify "(Door)"
- Rename morph door node to "Morph Ball Door to Crash Site"